specification/under_development/profiles/Base_Profile.md

```diff
@@ -147,13 +158,24 @@ declare void @__quantum__rt__result_record_output(%Result*, i8*)
 attributes #0 = { "entry_point" "qir_profile"="base_profile" "required_qubits"="2" "required_results"="2" }
 ```
 
-TODO: adjust text to reflect the split into different blocks
+For the sake of clarity, the code above does not contain any [debug
+symbols](https://releases.llvm.org/13.0.0/docs/tutorial/MyFirstLanguageFrontend/LangImpl09.html?highlight=debug%20symbols).
+Debug symbols contain information that is used by a debugger to related failures
```

Comment:
Missing verb between "debugger to" and "related"?

Reply:
This should have been "relate" instead of "related" and it is fixed now.